client/app/telecom/pack/xdsl/modem/router/dhcp/translations/Messages_fr_FR.xml

```diff
@@ -24,4 +24,6 @@
    <translation id="xdsl_modem_dhcp_submit_error" qtlid="385273">Oups ! Nous n'avons pas pu mettre à jour le DHCP.</translation>
    <translation id="xdsl_modem_dhcp_error" qtlid="385286">Oups ! Nous n'avons pas pu récupérer les informations relatives au DHCP.</translation>
    <translation id="xdsl_modem_dhcp_modal_title">Attention adresse IP refusée</translation>
+   <translation id="xdsl_modem_dhcp_modal_desc">L'adresse IP <strong>{{ ipUsed }}</strong> est un ancien serveur DNS OVH.</translation>
```

Comment:
Avoid using HTML in translations, add `strong` in the `translation-values` instead